packages/components/src/internal/util/Date.ts

```diff
@@ -197,10 +197,13 @@ function twoDigit(num: number): string {
     return '' + num;
 }
 
-// From a current date string, get the next date string
-// example, from "2022-02-02", return "2022-02-03"
-export function getNextDateStr(currentDateStr: string): string {
-    let nextDate = new Date(new Date(currentDateStr).getTime() + 60 * 60 * 24 * 1000); // add 24 hours
+// From a current date string, get the next N date string
+// example, from "2022-02-02", get next 1 day, return "2022-02-03"
+// example, from "2022-02-02", get next -1 day, return "2022-02-01"
+export function getNextDateStr(currentDateStr: string, ndays?: number): string {
+    const numberOfDays = ndays ?? 1;
```

Comment:
Does this do the right thing if `ndays` is 0?

Reply:
yep, jest test added